tests/integration_tests/test_cma.py

```diff
@@ -70,6 +72,29 @@ def test_init_default_values() -> None:
 
         assert isinstance(sampler._independent_sampler, optuna.samplers.RandomSampler)
 
+    @staticmethod
+    def test_warn_independent_sampling(capsys: _pytest.capture.CaptureFixture) -> None:
+        def objective(trial: Trial) -> float:
+            x = trial.suggest_categorical("x", ["a", "b"])
+            if x == "a":
+                return trial.suggest_float("y", 0, 1)
+            else:
+                return trial.suggest_float("z", 0, 1)
+
+        # We need to reconstruct our default handler to properly capture stderr.
+        optuna.logging._reset_library_root_logger()
+        optuna.logging.enable_default_handler()
+        optuna.logging.set_verbosity(optuna.logging.WARNING)
+
+        sampler = optuna.integration.PyCmaSampler(
+            warn_independent_sampling=True, n_startup_trials=0
+        )
+        study = optuna.create_study(sampler=sampler)
+        study.optimize(objective, n_trials=10)
+
+        _, err = capsys.readouterr()
+        assert err
```

Comment:
[Note] This is based on the test case of `TPESampler`.
https://github.com/optuna/optuna/blob/ceac5f13aa7c9ec9f5a7b08d59dec4c3bdbb2b2a/tests/samplers_tests/tpe_tests/test_sampler.py#L44-L62